lib/api/Healthcheck.js

```diff
@@ -84,6 +94,14 @@ class Healthcheck {
                 topics[this._repConfig.topic] = topicMd;
                 connections.isrHealth = this._checkISRHealth(topicMd);
             }
+            if (topicMd && this._isMissingISR(topicMd)) {
+                const error = {
+                    method: 'Healthcheck.getHealthcheck',
```

Comment:
Always return `Arsenal` style errors as they carry http code. I have fixed a bugs recently as the error was not Arsenal type. It's good to be consistent.

Reply:
I agree with that practice, although I am following the pattern of this method. There is a wrapper for this method that returns the `Arsenal` error [here](https://github.com/scality/backbeat/blob/development/8.0/lib/api/BackbeatAPI.js#L275). The way logging is done in that wrapper makes it so this method returns an error object like so.

Reply:
Ok..it's like we are accruing unnecessary tech debt. We need to clean that up sometime.

Reply:
I can refactor to make it consistent, but was hesitant to introduce changes beyond the scope of this healthcheck change.

Reply:
Opened a PR to come back and clean this up: https://github.com/scality/backbeat/pull/425.